docs/user_guide/HOWTO/Configure_Ssh.rst

```diff
@@ -1,118 +1,106 @@
 HOWTO Customize Settings for SSH
 ================================
 
-By default, including the **ssh** module will include ``ssh::server`` and
-``ssh::client``, configured with reasonable defaults for the OS and environment:
+By default, SIMP will include the ``simp-ssh`` module ``ssh`` class under all
+deployment scenarios. To exclude the SIMP ``ssh`` class, refer to
+:ref:`Disable SSH Managment <disable_ssh>`. To include specify the class for
+inclusion, add the following to :term:`Hiera`:
 
-.. code-block:: puppet
+.. code-block:: yaml
 
-   include 'ssh'
+   simp::classes:
+     - 'ssh'
 
-.. NOTE::
 
-   The examples below feature ``include 'ssh::server'`` and ``include
-   'ssh::client'``, but most SIMP scenarios already include them both via
-   ``ssh``.  So, for SIMP systems, you will customize parameter settings
-   for ``ssh::server`` and ``ssh::client`` via Hiera.
+The SIMP ``ssh`` class is configured to automatically include the
+``ssh::server`` and ``ssh::client`` classes. These classes manage the SSH
+daemon settings for incoming connections and the SSH client settings for
+outgoing connections respectively, and are configured with reasonable defaults
+for the OS and environment. To override this and **disable management** of one
+or both of these classes and manage them through some other mechanism, add the
```

Comment:
The sentence beginning with `To override this` has too many conjunctions for easy human parsing.  I suggest breaking it into two sentences, such as:
> If one or both of these class inclusions are disabled, the associated settings must be managed through some other mechanism. To **disable SIMP management** of one or both of these classes, add the following to Hiera: